test/run

```diff
@@ -4,5 +4,12 @@
 
 set -eu
 
+# run static code checks like pyflakes and pep8
```

Comment:
I think this would be a little more useful as `Makefile` rules, so that you can do `make codecheck` or something.

Reply:
Done